bottles/backend/wine/winecommand.py

```diff
@@ -487,6 +487,10 @@ def get_cmd(self, command, post_script: str = None, return_steam_cmd: bool = Fal
 
                 # Update command
                 command = f"{self.__get_gamescope_cmd(return_steam_cmd)} -- {gamescope_run}"
+                logging.info(f"Running Gamescope command: '{command}'")
+                logging.info(f"{gamescope_run} contains:")
+                with open(gamescope_run, "r") as f:
+                    print(f"\n{f.read()}\n")
```

Comment:
what's the purpose of this print?

Reply:
It's easier to debug and it shows exactly it's running.

Reply:
Can you replace it with `loggin` like other logs?

Reply:
Done.